site/en/guides/border-router/_toc.yaml

```diff
@@ -1,70 +1,59 @@
 toc:
 - title: Overview
   path: /guides/border-router/
-- title: Get Started
-  path: /guides/border-router/get-started
 
-- heading: Platforms
-- title: Android
-  path: /android/build-an-android-border-router
-- title: BeagleBone Black (Linux)
-  path: /guides/border-router/beaglebone-black
-- title: Raspberry Pi (Linux)
-  path: /guides/border-router/raspberry-pi
-- title: Espressif ESP32 Series (FreeRTOS)
-  path: /guides/border-router/espressif-esp32
-
-- heading: Setup
-- title: Build and Configuration
-  path: /guides/border-router/build
+- heading: Raspberry Pi Setup
+- title: Prepare
+  path: /guides/border-router/prepare
   step_group: otbr_setup
-- title: Web GUI
-  path: /guides/border-router/web-gui
+- title: Setup (Docker)
+  path: /guides/border-router/build-docker
+  step_group: otbr_setup
+- title: Setup (Native)
+  path: /guides/border-router/build-native
+  step_group: otbr_setup
+- title: Form Thread Network
+  path: /guides/border-router/form-network
   step_group: otbr_setup
 - title: mDNS Discovery
   path: /guides/border-router/mdns-discovery
   step_group: otbr_setup
 
+- heading: Other Platforms
+- title: Android
+  path: /android/build-an-android-border-router
+- title: Espressif ESP32 Series (FreeRTOS)
+  path: /guides/border-router/espressif-esp32
+
+- heading: Codelabs
+- title: OpenThread Border Router
+  status: external
```

Comment:
One last thing, can you remove the `status: external` from the OTBR codelab entries here and instead put them on the same 3 OTBR codelab entries in /guides/_toc.yaml?

On the site when one of these codelabs is selected it'll go to the Border Router tab where we want it to, but the external icon should be on the entries in the Get Started tab since it's going to take you somewhere else.